packages/loot-core/src/server/accounts/parse-file.js

```diff
@@ -57,6 +59,106 @@ async function parseCSV(filepath, options = {}) {
   return { errors, transactions: data };
 }
 
+function parseCAMT053Entries(data) {
+  let out = [];
+
+  data.forEach(element => {
+    let tmp = {};
+    tmp.date = element.ValDt.Dt._text;
+    
+    if (element.CdtDbtInd._text == "DBIT") {
+      tmp.amount = -looselyParseAmount(element.Amt._text);
+    } else {
+      tmp.amount = looselyParseAmount(element.Amt._text);
+    }
+    
+    if (typeof element.NtryDtls === 'undefined' || element.NtryDtls === null) {
```

Comment:
Those are some excellent property names they give you :) nit: you can use `element.NtryDtls == null` to check for both